templates/deployment.yaml

```diff
@@ -10,6 +10,8 @@ spec:
     matchLabels:
       app.kubernetes.io/name: {{ include "chart.name" . }}
       app.kubernetes.io/instance: {{ .Release.Name }}
+  strategy:
+    type: Recreate # better RollingUpdate, but needed for once mounted
```

Comment:
Could you provide more details? I have not experienced any problems yet with the default strategy (which is RollingUpdate I guess)

Reply:
The reason is also this lack of ReadWriteMany on some storage classes, which makes RollingUpdate impossible. We could make it configurable. In general, RollingUpdate is much better. 
I will make an option.

Reply:
Ok, without ReadWriteMany it will not be possible, of course